> thanks for your contribution, would you mind having a look at the failing tests?

it's a lint failed at `smart_importer/predictor.py:57:4: R0917: Too many positional arguments (6/5) (too-many-positional-arguments)` , how would you like to resolve it? disable it for this function?

--- a/smart_importer/predictor.py
+++ b/smart_importer/predictor.py
@@ -54,7 +54,7 @@ class EntryPredictor:
     weights: dict[str, float] = {}
     attribute: str | None = None
 
-    def __init__(
+    def __init__(#pylint: disable=too-many-arguments
         self,
         predict: bool = True,
         overwrite: bool = False,

--- a/smart_importer/predictor.py
+++ b/smart_importer/predictor.py
@@ -54,7 +54,7 @@ class EntryPredictor:
     weights: dict[str, float] = {}
     attribute: str | None = None
 
-    def __init__(  # pylint: disable=too-many-arguments
+    def __init__(  # pylint: disable=too-many-positional-arguments,too-many-arguments
         self,
         predict: bool = True,
         overwrite: bool = False,

--- a/smart_importer/predictor.py
+++ b/smart_importer/predictor.py
@@ -54,7 +54,7 @@ class EntryPredictor:
     weights: dict[str, float] = {}
     attribute: str | None = None
 
-    def __init__(#pylint: disable=too-many-arguments
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         predict: bool = True,
         overwrite: bool = False,

